Assets/Mirror/Components/NetworkTransformChild.cs

```diff
@@ -6,13 +6,13 @@ namespace Mirror
     /// A component to synchronize the position of child transforms of networked objects.
     /// <para>There must be a NetworkTransform on the root object of the hierarchy. There can be multiple NetworkTransformChild components on an object. This does not use physics for synchronization, it simply synchronizes the localPosition and localRotation of the child transform and lerps towards the recieved values.</para>
     /// </summary>
-    [AddComponentMenu("Network/NetworkTransformChild")]
+    [AddComponentMenu("Network/NetworkTransformChildExperimentalExperimental")]
     [HelpURL("https://mirror-networking.com/docs/Components/NetworkTransformChild.html")]
     public class NetworkTransformChild : NetworkTransformBase
     {
         [Header("Target")]
         public Transform target;
 
-        protected override Transform targetComponent => target;
+        protected override Transform targetTransform => target;
```

Comment:
renaming this will break people's projects.
what's wrong with targetComponent?

Reply:
The entire component is a breaking change.  The type is Transform and should've been named this way all along.